compiler/src/iree/compiler/Codegen/Dialect/VectorExt/IR/VectorExtAttrs.cpp

```diff
@@ -270,6 +270,64 @@ NestedLayoutAttr NestedLayoutAttr::get(MLIRContext *context,
                                threadStrides);
 }
 
+NestedLayoutAttr NestedLayoutAttr::get(
```

Comment:
This can fail right? It's possible that we have an incomplete layout or a mismatch?

Reply:
I did not find good error handling in vector layout analysis code; most places had asserts
should I assert that ?

Reply:
Do we need this builder? We already have getRecombindedLayout. The failure more in the analysis is that if a layout cannot be inferred, we just do not propagate/enforce it.

Reply:
ah .. I can inline this builder into getRecombindedLayout if thats what you meant ?

Reply:
yes

Reply:
inlined and added checks.